src/main/resources/locales/en/pending-subscriptions.json

```diff
@@ -1,12 +1,14 @@
 {
     "title": "Confirm your email subscriptions",
+    "title_court_subscription": "Your email subscriptions",
     "removeText": "Remove",
     "table": {
         "header1": "Case name",
         "header2": "Party name(s)",
         "header3": "Reference number",
         "header4": "Actions"
     },
+    "buttonText_court_subscription": "Continue",
```

Comment:
title_court_subscription and buttonText_court_subscription need to be added to the welsh language resource too

Reply:
added